manage.py

```diff
@@ -118,7 +118,7 @@ def print_flush(msg):
     print_every = 1000
     print_flush("Adding waterpoints. Please be patient.")
 
-    with open(filename) as f:
+    with open(filename, 'rU') as f:
```

Comment:
This is just to make life easier when exporting to csv from Excel (what you typically do as part of the workflow). Adding 'U' ensures those csv's can be easily read too.